api/users/serializers.py

```diff
@@ -11,6 +11,29 @@
 )
 from api.base.utils import absolute_reverse
 
+class PublicFilesSerializer(JSONAPISerializer):
```

Comment:
Yeah you might take a look at this as well, if you haven't already https://github.com/CenterForOpenScience/osf.io/pull/5996, it's on hold awaiting Jeff's go head. Forcing guid creation is a heavily discussed issue. Making it dependent on a PublicFIles endpoint, would limit spamming guid creation, but It might effect Preprints who also need to force guids. I'll inquire what there needs are.